numpy/random/tests/test_random.py

```diff
@@ -693,6 +693,97 @@ def test_multivariate_normal(self):
         assert_raises(ValueError, np.random.multivariate_normal, mean, cov,
                       check_valid='raise')
 
+    def test_multivariate_hypergeometric(self):
```

Comment:
This is big enough that I'd rather split it up into separate tests in their own class. Note that you can use hashes for repeatability if you want to test larger numbers of samples.

Reply:
Looks like a some trailing white space has crept into master, if that is fixable here that would be nice.